BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig

```diff
@@ -1,3 +1,13 @@
 <div class="flashbag">
-    <%= html %>
+    <% if (typeof errors.error != 'undefined') { %>
```

Comment:
this code is duplicate from Braincrafted and will not be synchronized on update. The reason why i did the render with Braincrafted.